lib/inputs/rhel/static.ts

```diff
@@ -0,0 +1,27 @@
+import { ExtractAction, ExtractedLayers } from "../../extractor/types";
+import { streamToJson } from "../../stream-utils";
+
+export const getRedHatReposContentAction: ExtractAction = {
+  actionName: "redhat-content-manifests",
+  filePathMatches: (filePath) => isRedHatContentManifest(filePath),
+  callback: streamToJson,
+};
+export function getRedHatReposFromExtractedLayers(
+  extractedLayers: ExtractedLayers,
+) {
+  const reposByLayer = {};
+  for (const filePath in extractedLayers) {
+    if (isRedHatContentManifest(filePath)) {
+      const content = extractedLayers[filePath][
+        "redhat-content-manifests"
+      ] as any;
+      const layerIndex = content.metadata.image_layer_index;
+      reposByLayer[layerIndex] = content.content_sets;
```

Comment:
I think I'd like this object to more explicitly mention repositories, so then it's going to be clearer to read the code when doing all the matching stuff, wdyt??